timelock-agent/src/main/java/com/palantir/timelock/config/PaxosRuntimeConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.timelock.config;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.base.Preconditions;
+
+@JsonDeserialize(as = ImmutablePaxosRuntimeConfiguration.class)
+@JsonSerialize(as = ImmutablePaxosRuntimeConfiguration.class)
+@Value.Immutable
+public interface PaxosRuntimeConfiguration extends TimeLockAlgorithmRuntimeConfiguration {
+    @JsonProperty("ping-rate-in-ms")
+    @Value.Default
+    default long pingRateMs() {
+        return 5000L;
+    }
+
+    @JsonProperty("maximum-wait-before-proposal-in-ms")
+    @Value.Default
+    default long maximumWaitBeforeProposalMs() {
+        return 1000L;
+    }
+
+    @JsonProperty("leader-ping-response-wait-in-ms")
+    @Value.Default
+    default long leaderPingResponseWaitMs() {
+        return 5000L;
+    }
```

Comment:
I didn't change the new methods names but would be happy to. I changed the property names because internally we've decided everything should be snake-case.
Oh and I guess in some instances I changed things to be like `-in-ms` but am happy to undo that change, or make it something different entirely.